.travis.yml

```diff
@@ -1,9 +1,18 @@
+sudo: true
 language: node_js
-node_js:
-  - 10
-  - 8
-  - 6
+
+matrix:
+  include:
+    - node_js: "10"
+    - node_js: "8"
+    - node_js: "6"
+      env: RUN_RECIPES=1
 before_install:
   - npm install --global npm@latest
+install:
+  -  npm install -s --no-progress
+before_script:
+  - npm run tabtab-install
+  - source /home/travis/.bashrc
```

Comment:
As far as I can tell (now that I worked a bit on this, sorry for my previous answers), and unless I didn't understood the expected installation step, I'm not sure you need this anymore. It'll be done by `yo completion` and up to the user to choose when and how to setup the bridge in its bashrc (or zshrc, or fish config).

The tests don't really test the autocompletion but the actual output of yo-completer (on `yo completion -- yo <command or options>`)